diff --git a/include/paf/paf_types.h b/include/paf/paf_types.h
index 74286a6..951b24f 100644
--- a/include/paf/paf_types.h
+++ b/include/paf/paf_types.h
@@ -11,6 +11,10 @@
 extern "C" {
 #endif
 
+#ifndef __declspec
+#define __declspec(__v__)
+#endif
+
 // typedef int64_t off_t;
 
 typedef enum ImageMode {
diff --git a/include/paf/std/wchar.h b/include/paf/std/wchar.h
index c2cdd20..9289b98 100644
--- a/include/paf/std/wchar.h
+++ b/include/paf/std/wchar.h
@@ -6,6 +6,7 @@
 #define _VDSUITE_USER_PAF_STD_WCHAR_H
 
 #include <../../wchar.h>
+#include <stdarg.h>
 
 #ifdef __cplusplus
 extern "C" {
diff --git a/lib/SceAppSettings.emd b/lib/SceAppSettings.emd
index 5ddb992..6b6ccc9 100644
--- a/lib/SceAppSettings.emd
+++ b/lib/SceAppSettings.emd
@@ -17,7 +17,7 @@ Library: SceAppSettings function: _ZN3sce11AppSettings10InitializeEv nidvalue: 0
 Library: SceAppSettings function: _ZN3sce11AppSettings9SetStringEPKcS2_ nidvalue: 0x9D20A196
 Library: SceAppSettings function: _ZN3sce11AppSettings9GetStringEPKcS2_jS2_ nidvalue: 0x1E1BE36B
 Library: SceAppSettings function: _ZN3sce11AppSettings9TerminateEv nidvalue: 0x1C135375
-Library: SceAppSettings function: _ZN3sce11AppSettings7GetBoolEPKcbb nidvalue: 0x67FD397D
+Library: SceAppSettings function: _ZN3sce11AppSettings7GetBoolEPKcPbb nidvalue: 0x67FD397D
 Library: SceAppSettings function: _ZN3sce11AppSettings7SetBoolEPKcb nidvalue: 0x562213F4
 
 Library: SceAppSettings function: _ZN3sce11AppSettings7Element11GetPropertyEPKc nidvalue: 0xEECA5057
