diff --git a/lib/SceAppSettings.emd b/lib/SceAppSettings.emd
index 5ddb992..6b6ccc9 100644
--- a/lib/SceAppSettings.emd
+++ b/lib/SceAppSettings.emd
@@ -17,7 +17,7 @@ Library: SceAppSettings function: _ZN3sce11AppSettings10InitializeEv nidvalue: 0
 Library: SceAppSettings function: _ZN3sce11AppSettings9SetStringEPKcS2_ nidvalue: 0x9D20A196
 Library: SceAppSettings function: _ZN3sce11AppSettings9GetStringEPKcS2_jS2_ nidvalue: 0x1E1BE36B
 Library: SceAppSettings function: _ZN3sce11AppSettings9TerminateEv nidvalue: 0x1C135375
-Library: SceAppSettings function: _ZN3sce11AppSettings7GetBoolEPKcbb nidvalue: 0x67FD397D
+Library: SceAppSettings function: _ZN3sce11AppSettings7GetBoolEPKcPbb nidvalue: 0x67FD397D
 Library: SceAppSettings function: _ZN3sce11AppSettings7SetBoolEPKcb nidvalue: 0x562213F4
 
 Library: SceAppSettings function: _ZN3sce11AppSettings7Element11GetPropertyEPKc nidvalue: 0xEECA5057
